### DISCUSSION TITLE
[iwxxm-collect] conflicting element names in v3.1 (rc)

### DISCUSSION THREAD

**Discussion by kcris** [wmo-im/iwxxm#280] [Q&A] [answered]
Trying to consume the 2021-2RC1 iwxxm schemas 
(via java generator) 
I noticed that when loading iwxxm-collect.xsd conflicting element names are detected

for example

this is part of wafssigwxfc.xsd
```xml
	<element name="Volcano" type="iwxxm:VolcanoType" substitutionGroup="gml:AbstractGML"> ...
```

and this is part of phenomena.xsd
```xml
        <element name="Volcano" type="metce:VolcanoType" substitutionGroup="gml:AbstractFeature"> ...
```

edit:
namespace is not the same so this is only a problem with the generating side
closing as invalid

**Comment by blchoy** [comment, marked as answer]
They are different.

wafssigwxfc.xsd is part of IWXXM while phenomena.xsd is part of METCE, and they are under different namespaces (viz http://icao.int/iwxxm/2021-2 and http://def.wmo.int/metce/2013).

You are right as the path of the java-bindings should be properly managed.
